snapcraft/plugins/v2/cmake.py

```diff
@@ -84,6 +90,7 @@ def _get_cmake_configure_command(self) -> str:
             '"${SNAPCRAFT_PART_SRC_WORK}"',
             "-G",
             f'"{self.options.cmake_generator}"',
+            "${SNAPCRAFT_CMAKE_ARGS}",
```

Comment:
Can't we just use a `cmake-parameters` in the extension?

Reply:
The strict goal for extensions is to always extend snapcraft.yaml, if we can eventually pull it off, would be good